Software-Engineering.md

```diff
@@ -133,6 +133,24 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I can confidently and charismatically pitch ideas, positively influencing my team to take decisive action based on reaching consensus.
 - I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
 
+## Principal Software Engineer
+
+> I am an influential Engineering [leader](Leadership.md) who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus.
+
+### I have a knack for uncovering important problems and opportunities that are critical to the long-term success of Octopus
+
+- I have a deep understanding of Octopus products and our strategy
+- I identify significant problems, inhibitors and opportunities, and advocate for solutions that align our strategy and Engineering efforts
+- I collaborate with others to articulate solutions, positively influence and convince that we should fund their implementation
+- My contributions go beyond patching or extending existing solutions, which may not be fit for purpose anymore. They open up whole new areas of strategic possibility, to take our Engineering team, and Octopus as a whole, to the next level.
+
+### I am trusted to take care of a large, critical or complex strategic area or product
```

Comment:
What does "take care of" mean?

Reply:
I was reflecting on the new team shape for the actions team: PSE + EM + PM forming the leadership group for that team. It feels like this team would be collectively responsible for the outcomes of that team. I think that is what "take care of" means in this context. Is "accountable" a better word?

```suggestion
### I am accountable for a large, critical or complex strategic area or product
```

Reply:
I like this @TomPeters - if we combine it with one of @michaelnoonan's 🌱 's below, and add a bit of Hemmingway, we get:

```suggestion
### I am the custodian of an area of strategic importance to Octopus
```